src/components/Results.tsx

```diff
@@ -9,16 +9,22 @@ import { HEADER } from 'src/config/componentSizes';
 import { useSearchParams } from 'react-router-dom';
 import { accommodationEventTransform } from '../hooks/useAccommodationsAndOffers.tsx/helpers';
 
+export enum ResultsMode {
+  map,
```

Comment:
what is the difference between `map` and `list` in terms of UI?

Reply:
.

Reply:
results on map or in list for monile